### DISCUSSION TITLE
It's official website is no friendly to a truly newbie!!

### DISCUSSION THREAD

**Discussion by lymtu** [vercel/next.js#66559] [Show and tell]
Those example in website may be suited to someone had experience in, for example:
In office site, it no show me how to create pages files with scaffold, 

so, i guess and try `app/test/page.tsx`

and then i try to use `getStaticProps`, a error boom
![PixPin_2024-06-05_17-44-00](https://github.com/vercel/next.js/assets/134980038/5963b24f-af8b-4088-9d71-a008c204bd24)

i am truly wonder why many frame do like this, 

or maybe it's me?

**Comment by icyJoseph** [comment]
Hi,

`getStaticProps` is part of the Pages Router family of functions. You have `app/test/page.tsx`, which indicates you are in the App Router stack, where `getStaticProps` doesn't exist. Please follow the documentation.

To get data in App Router, you can just use an async server component, and `await` the data within the component, https://nextjs.org/docs/app/building-your-application/data-fetching/fetching-caching-and-revalidating:

<img width="1370" alt="Screenshot 2024-06-05 at 11 52 25" src="https://github.com/vercel/next.js/assets/21013447/a689fe96-874c-49d6-9b3d-9386e4bc94d7">

**Reply by lymtu** [reply]
Thank you for your reply,maybe I was too hasty, the path introduction of this introductory document made me a little confused, maybe I should try to use the most people do not use scaffolding, in the root directory with 'pages/' instead of 'app/'
